serverless/event_workflows/serverless-sinkbinding.adoc

```diff
@@ -10,10 +10,30 @@ SinkBinding is used to connect event producers, or _event sources_, to an event
 
 [NOTE]
 ====
-Both of the following procedures require you to create YAML files.
+The following procedures require you to create YAML files.
 
 If you change the names of the YAML files from those used in the examples, you must ensure that you also update the corresponding CLI commands.
 ====
 
+To use a SinkBinding, cluster administrators must label the namespace configured in the SinkBinding with `bindings.knative.dev/include:"true"`:
```

Comment:
This information should be in a module, maybe serverless-sinkbinding-kn.adoc, not the assembly. I think that labeling the namespace might be a separate step, and I'm not sure if the info about the label should be a note on existing yaml. (I only see it in the release notes, so I'm not sure where.)

Reply:
Added this whole section to both the modules, lmk if that's better :slightly_smiling_face: I wasn't sure about duplicating it but it needs to be in both the yaml and kn modules if it's not in the assembly.